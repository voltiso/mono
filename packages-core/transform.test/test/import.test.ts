// â €â“¥ 2022     ðŸŒ©    ðŸŒ©     â €   â €
// â €         ðŸŒ© VÍ›oÍ›Í›Í›ltÍ›Í›Í›iÍ›Í›Í›Í›soÍ›Í›Í›.comâ €  â €â €â €

import * as fs from 'node:fs/promises'

describe('import', () => {
	it('works', async () => {
		expect.hasAssertions()

		const file = await fs.readFile('./dist/cjs/import.d.ts')
		expect(file.toString()).toMatchSnapshot()
	})
})
