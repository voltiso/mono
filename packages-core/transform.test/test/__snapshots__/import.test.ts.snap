// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`import works 1`] = `
"import type { ExternalInterface, IncludesExternal } from './_/external-stuff';
export declare type A = /* IncludesExternal */ { ex: ExternalInterface | { a: 1; }; } & ExternalInterface;
//# sourceMappingURL=import.d.ts.map"
`;
